packages/json/src/browser/json-client-contribution.ts

```diff
@@ -95,15 +84,21 @@ export class JsonClientContribution extends BaseLanguageClientContribution {
     }
 
     protected onReady(languageClient: ILanguageClient): void {
+        super.onReady(languageClient);
         // handle content request
         languageClient.onRequest('vscode/content', async (uriPath: string) => {
             const uri = new URI(uriPath);
             const resource = await this.resourceProvider(uri);
             const text = await resource.readContents();
             return text;
         });
-        super.onReady(languageClient);
-        setTimeout(() => this.initializeJsonSchemaAssociations());
```

Comment:
@svenefftinge I've removed it because it adds a lot of duplicate schemas on each connection. Calling it once from the constructor should be enough?